### ISSUE TITLE
Prometheus metrics

### ISSUE THREAD

**Issue by deviantony** [deviantony/docker-CRN#26]
We need to find out which _Prometheus_ metrics we need to process in order to achieve container down-scale, up-scale...

These metrics will be retrieved in the _central-core_ component via the _Prometheus_ API.

**Comment by deviantony** [comment]
I've used the gatling load tests to determine which metrics we could exploit:
- _haproxy_frontend_current_sessions { frontend="http-front" }_ and _haproxy_frontend_current_session_rate { frontend="http-front" }_  : the current number of sessions and the current session rate (requests/seconds) on the "http-front" frontend. Using gatling basic tests I've been able to get an average number of sessions of 300 and an average rate of 500 requests/sec during a short period of time. We could use these metrics to scale dynamically.
- _consul_catalog_service_node_healthy { service="myService" }_: value indicating the health of the service (1 for OK, 0 for KO). We could use this metric to manage application respawn.
